packages/pymedphys_databases/src/pymedphys_databases/deliverydata/core.py

```diff
@@ -0,0 +1,7 @@
+from pymedphys_base.deliverydata import DeliveryData as DeliveryDataBase
+from pymedphys_fileformats.deliverydata import DeliveryDataLogfile
+from pymedphys_mudensity.deliverydata import DeliveryDataMuDensity
+
+
+class DeliveryDataDatabases(DeliveryDataMuDensity, DeliveryDataLogfile, DeliveryDataBase):  # type: ignore
```

Comment:
There are quite a few methods that belong on here. ... But that's not a job for this pull request.

Reply:
... the name is awkward also ... Will fix that at some point.